service/assessment/assessment.go

```diff
@@ -377,75 +387,24 @@
 }
 
 // initEvidenceStoreStream initializes the stream to the Evidence Store
-func (s *Service) initEvidenceStoreStream(additionalOpts ...grpc.DialOption) error {
+func (s *Service) initEvidenceStoreStream(URL string, additionalOpts ...grpc.DialOption) (stream evidence.EvidenceStore_StoreEvidencesClient, err error) {
```

Comment:
Isn't `URL` and `s.evidenceStoreAddress` the same?

Reply:
See comment from garuppel. It needs to have the URL otherwise it is not an initfunc